src/components/UserProfile/TeamsAndProjects/UserTeamsTable.jsx

```diff
@@ -37,7 +37,7 @@ const UserTeamsTable = props => {
     <div>
       <div className="teamtable-container desktop">
         <div className="container" style={{paddingLeft: '4px', paddingRight: '4px'}}>
-          {props.canEditVisibility && (
+          {(props.canEditVisibility || canToggleInvisibility) && (
```

Comment:
```suggestion
          {(canToggleInvisibility) && (
```
Should replace/remove `canEditVisibility` from props and include logic in `canToggleInvisibility`

Reply:
Should also replace at line 147.